### ISSUE TITLE
Argument to write 'Bag-Size' to created bags?

### ISSUE THREAD

**Issue by axfelix** [LibraryOfCongress/bagit-python#142]
Hi,

Should there be an argument to pass to `make_bag()` that adds a `Bag-Size` property automatically? That seems like an appropriate time to hook this in rather than trying to calculate and append a standard property after the fact.

**Comment by nkrabben** [comment]
Does the oxum meet your needs for a Bag-Size property? It's in bag-info.txt in the form ####.### where the first part is the total bytes in the data folder and the second is the total number files in the data folder.

**Comment by axfelix** [comment]
Yup, I think so! Just looking to have that named element as I believe it's mentioned in the spec.
